### DISCUSSION TITLE
bad CPU type in executable: vector error

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#17254] [Q&A]
Hi.
i installed vector dev binary on mac os (air m2 2022)
```
brew tap vectordotdev/brew && brew install vector
```
but i got this error

```
~ vector 

zsh: bad CPU type in executable: vector
```

**Comment by spencergilbert** [comment]
Unfortunately at this time we're not publishing artifacts for ARM Macs, that is tracked in https://github.com/vectordotdev/vector/issues/10193. You can run the x86 binary with rosetta, or build Vector yourself locally for a native binary.

**Reply by LinTechSo** [reply]
thanks for the reply.
